src/uwtools/utils/tasks.py

```diff
@@ -59,18 +79,35 @@ def file(path: Path, context: str = ""):
 
 
 @task
-def filecopy(src: Path, dst: Path):
+def filecopy(src: Union[Path, str], dst: Union[Path, str]):
```

Comment:
Similar to the updated `existing()` above, retain the old logic for local filesystem items, have `requests` do a `GET` on HTTP objects, and raise an exception for unsupported schemes.